SingularityUI/app/components/logs/FileNotFound.jsx

```diff
@@ -1,8 +1,36 @@
-import React from 'react';
+import React, {PropTypes} from 'react';
+
+const RENAMED_FILES = {
+  'service.log': 'tail_of_finished_service.log'
```

Comment:
These filenames aren't constant, but configurable. You can get the values from `config.runningTaskLogPath` and `config.finishedTaskLogPath`.